src/engine/strat_engine/thinpool/thinpool.rs

```diff
@@ -42,12 +43,12 @@ use super::filesystem::{FilesystemStatus, StratFilesystem};
 use super::mdv::MetadataVol;
 use super::thinids::ThinDevIdPool;
 
-pub const DATA_BLOCK_SIZE: Sectors = Sectors(2 * IEC::Ki);
-const DATA_LOWATER: DataBlocks = DataBlocks(512);
-const META_LOWATER_FALLBACK: MetaBlocks = MetaBlocks(512);
+pub const DATA_BLOCK_SIZE: Sectors = Sectors(2 * IEC::Ki); // 1 MiB
+pub const DATA_LOWATER: DataBlocks = DataBlocks(4096); // 2 GiB
+const META_LOWATER_FALLBACK: MetaBlocks = MetaBlocks(512); // 2 KiB
 
-const INITIAL_META_SIZE: MetaBlocks = MetaBlocks(4 * IEC::Ki);
-const INITIAL_DATA_SIZE: DataBlocks = DataBlocks(768);
+const INITIAL_META_SIZE: MetaBlocks = MetaBlocks(262144); // 1 GiB
+const INITIAL_DATA_SIZE: DataBlocks = DataBlocks(768); // 4 GiB
```

Comment:
Comment doesn't match code.

If upped to 4GiB, this also will run afoul of the 1 GiB MIN_DEV_SIZE. I'd recommend taking the "request" approach that #1260 took -- i.e. if the pool is initially big enough, allocate 4GiB, but if it isn't just allocate whatever remains. (Feel free to lift code directly from that PR if it helps.)

Reply:
Ok.  I lifted code from #1260 as you suggested.